src/interface/src/app/plan/create-scenarios/create-scenarios.component.spec.ts

```diff
@@ -158,36 +161,40 @@ describe('CreateScenariosComponent', () => {
       MatButtonHarness.with({ text: /GENERATE/ })
     );
     component.formGroups[0]
+      .get('scenarioName')
+      ?.setValue('scenarioName');
+    component.formGroups[1]
       .get('selectedQuestion')
       ?.setValue(defaultSelectedQuestion);
-    component.formGroups[1].get('physicalConstraintForm.maxSlope')?.setValue(1);
-    component.formGroups[1]
+    component.formGroups[2].get('physicalConstraintForm.maxSlope')?.setValue(1);
+    component.formGroups[2]
       .get('physicalConstraintForm.minDistanceFromRoad')
       ?.setValue(1);
-    component.formGroups[1].get('physicalConstraintForm.maxArea')?.setValue(1);
+    component.formGroups[2].get('physicalConstraintForm.maxArea')?.setValue(1);
     component.generatingScenario = false;
     fixture.detectChanges();
 
     expect(await buttonHarness.isDisabled()).toBeFalse();
   });
 
-  it('update plan state when "identify project areas" form inputs change', () => {
-    const generateAreas = component.formGroups[2].get('generateAreas');
-    const uploadedArea = component.formGroups[2].get('uploadedArea');
+  // TODO Re-enable when support for uploading project areas in implemented
```

Comment:
are this tests no longer valid? should we remove them or re write them?

Reply:
I thought it was best to keep this one around since I believe the test won't need to be changed - it should just have to be re-enabled once users are allowed to upload their own project areas again. Let me know if you would rather I just delete it for now (since the next time it's needed will likely be after we're gone)